_posts/2025-07-02-this-month-in-servo.md

```diff
@@ -0,0 +1,534 @@
+---
+layout:     post
+tags:       blog
+title:      "This month in Servo: ?????"
+date:       2025-07-02
+summary:    awooga awooga
+categories:
+---
+
+June was Servo’s busiest month in years, with **349 pull requests** landing in our nightly builds!
+
+<canvas id="_stats"></canvas>
+
+Servo now supports **viewport meta** tags (@shubhamg13, @xiaochengh, #35901), the **exportparts** attribute (@simonwuelker, #37345), the **‘::part()’ selector** (@simonwuelker, #37307), and several new web API features:
+
+- **ImageBitmap** (@tharkum, #37025, #37397, #37488, #37560, #37634)
+- the **supports() method** on **HTMLScriptElement** (@pewsheen, #37313)
+- the **lang()** and **id()** functions in **XPath** (@vlindhol, #34594)
+- the **&lt;hr size> attribute** (@simonwuelker, #37211)
+- barebones **IndexedDB** support (@arihant2math, @rasviitanen, @jdm, #33044, #37652, #37653)
```

Comment:
Should it be noted this is gated by the `dom_indexeddb_enabled` preference?

Reply:
great catch! done in d012051fe6beeb95f0af23e817f96cb26637a87d